packages/terra-date-picker/src/terra-dev-site/test/date-picker/DatePickerOnChangeRaw.test.jsx

```diff
@@ -13,7 +13,7 @@ class DatePickerOnChange extends React.Component {
       iSO: '',
       inputValue: '',
       isCompleteValue: 'No',
-      isValidValue: 'Yes',
+      isValidValue: 'No',
```

Comment:
why this change?

Reply:
I'll revert it. It resulted from some confusion as to why a test would be checking to verify that `04/01/2019` is an invalid date. There's a little bit of ambiguity in referring to an excluded date as an invalid date, so I had things mixed around a bit. :)